+dj/key_hash.m

```diff
@@ -0,0 +1,7 @@
+% dj.key_hash - key hashing function for populate jobs, etc
```

Comment:
Since we are adding to the code base, would you add a minimal test for taking the hash to maintain the coverage?

Also, would you remove this and add it properly in the function so that it works with `help(...)`? See [here](https://github.com/datajoint/datajoint-matlab/blob/755ea67c0d348ddbc4d84736e0412667ffa627da/%2Bdj/config.m#L2-L27) for an example.